main.rb

```diff
@@ -51,13 +28,13 @@ def h(text)
 end
 
 post '/memos' do
-  memo = { 'id' => build_id, 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }
-  save_memo(memo['id'], memo)
-  redirect "/memos"
+  memo = { 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }.values
```

Comment:
```suggestion
  memo = { 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }
```

配列にすると「順番に意味を持つ配列」になってしまうのでハッシュのままにした方がベターです。
https://qiita.com/jnchito/items/fb4ed2238ee3404b4403

もちろん、SQLのバインドパラメータに渡すときは予め配列になっている方が便利なのですが、`Memo.create`を呼び出す側としてはそんな内部の事情は知ったこっちゃないので、あくまで`Memo.create`メソッドを外から見たときにどういうAPI設計（＝引数と戻り値の設計）にすべきかを考えるようにしましょう。